### DISCUSSION TITLE
Immich added to Badges4-README.md-Profile

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#14678] [Show and tell]
I went ahead and added an immich badge to the popular badge repo [Badges4-README.md-Profile](https://github.com/alexandresanlim/Badges4-README.md-Profile). If owners or maintainers want this changed please do 👍🏻 

Here is the PR I cut: https://github.com/alexandresanlim/Badges4-README.md-Profile/pull/740

**Comment by danieldietzler** [comment]
Hey, thanks! I personally find that pretty neat. Would you able to actually have all five colors in the petal, instead of just red?

**Reply by coltenkrauter** [reply]
I tried, but I don't think it is supported by [Shields.io](https://shields.io/), but I expect we could use some other badge maker to achieve that. I agree that would be awesome.

**Reply by danieldietzler** [reply]
Looks like there aren't actually many badges that have this, https://camo.githubusercontent.com/12ec0e487152f2453d1428ee5e45e2e95b5381872a3117018c42cd26ab88f1ec/68747470733a2f2f637573746f6d2d69636f6e2d6261646765732e64656d6f6c61622e636f6d2f62616467652f636f6d65742532306d6c2d3236326333653f7374796c653d666f722d7468652d6261646765266c6f676f3d6c6f676f5f636f6d65745f6d6c266c6f676f436f6c6f723d7768697465 does though for instance. So it must be doable somehow 😅
